supabase/functions/deploy-community-config/index.ts

```
@@ -0,0 +1,2370 @@
+  const bytes = crypto.getRandomValues(new Uint8Array(length));
+  return Array.from(bytes, (byte) => alphabet[byte % alphabet.length]).join("");
+};
+
+const buildBlankSubdomain = async (
+  supabase: ReturnType<typeof createClient>,
+  displayName: string
+) => {
+  const baseSlug = slugifyDisplayName(displayName || "space");
+  let candidate = `${baseSlug}.blank.space`;
+
+  while (true) {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)